bench/wt2853_perf/main.c

```diff
@@ -107,8 +107,8 @@ main(int argc, char *argv[])
     testutil_make_work_dir(opts->home);
     testutil_progress(opts, "start");
 
-    testutil_check(
-      wiredtiger_open(opts->home, NULL, "create,cache_size=1G,statistics=(all)", &opts->conn));
+    testutil_check(wiredtiger_open(opts->home, NULL,
+      "create,cache_size=1G,statistics=(all),statistics_log=(json,on_close,wait=5)", &opts->conn));
```

Comment:
Is `5` on purpose? The other changes use `1`.

Reply:
perf test should have a longer interval I think.

Reply:
test format is also set as 5